android/app/src/main/AndroidManifest.xml

```diff
@@ -20,6 +20,7 @@
     <activity
       android:name=".MainActivity"
       android:configChanges="orientation|keyboardHidden|keyboard|screenSize|smallestScreenSize|locale|layoutDirection|fontScale|screenLayout|density|uiMode"
+      android:exported="true"
```

Comment:
What's the reason for adding this?

Reply:
Resolve conflict when merging manifest files.

Reply:
Ah I see, this became mandatory after bumping the target sdk